run.sh

```diff
@@ -63,6 +63,7 @@ if [ "$skip_preprocess" = false ]; then
   python arkit_utils/undistort_images/undistort_image.py --input_base ${input_base_path}
 
   echo "2. Transform ARKit mesh to point3D"
+  cp ${input_base_path}/../scene.obj ${input_base_path}
```

Comment:
just a minor advice, maybe we can add a validation of the path here

Reply:
solved in latest commit